### ISSUE TITLE
Feature requests: Creating dropdown detail cells or a detail object

### ISSUE THREAD

**Issue by pashute** [rstaib/jquery-bootgrid#189]
1. Is it possible (or easy) to have a details row or details cell open with a bootgrid dropdown?

2. Would you consider please creating a details component that is connected to the current selected object.

**Comment by rstaib** [comment]
Do you want to have a ui component inside a cell? In that case see the [command button](http://www.jquery-bootgrid.com/Examples#command-buttons) example.

**Comment by pashute** [comment]
No. I want to have "details"  of the clicked "row" (from a details table in the db)
It is a common thing for grids to have.

**Comment by rstaib** [comment]
Is it a 1:1 relation and should it be part of the request you're doing to get the rows?

**Comment by pashute** [comment]
​​
In most cases (as found in most "data grids") there is the possibitity for
a 1 to many details rows that open up either by clicking a "hierarchy icon"
such as a plus sign, or by clicking on the row.

It is an accordion like behavior and may be replaced with a modal that
shows the details table in  it.

For a 1:1 details usually a dropdown from the cell but which can span the
cell left and right border and extend to the end of the table.

**Comment by wjk1982** [comment]
+1
